bugbug/bug_features.py

```diff
@@ -131,6 +134,12 @@ def __call__(self, bug):
         return any(bug['creator_detail']['email'].endswith(domain) for domain in ['@mozilla.com', '@mozilla.org'])
 
 
+class delta_request_merge(object):
+    def __call__(self, bug):
+        uplift_request_datetime = datetime.strptime(bug['history'][0]['when'], '%Y-%m-%dT%H:%M:%SZ').replace(tzinfo=timezone.utc)
```

Comment:
This is the date of the first modification to the bug, we should get the date of the first uplift request instead.